ambry-named-mysql/src/integration-test/java/com/github/ambry/named/MySqlNamedBlobDbIntegrationTest.java

```diff
@@ -210,6 +230,79 @@ public void testExpiredBlobs() throws Exception {
         namedBlobDb.get(account.getName(), container.getName(), blobName).get());
   }
 
+  /**
+   * Test behavior with stale blob cleanup
+   */
+  @Test
+  public void testCleanupStaleBlobs() throws Exception {
+    Calendar calendar = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
+    calendar.add(Calendar.MONTH, -2);
```

Comment:
this should be in the configuration. It's better to get the retention time from the configuration so that you it's the same in the named blob db.

Reply:
Done